public/index.php

```diff
@@ -13,5 +13,5 @@
     $app->run();
 } catch (Exception $e) {
     // Handle any exceptions that occur during the application run
-    die('An error occurred: ' . $e->getMessage());
+    die('<pre>An error occurred: ' . $e->getMessage() . PHP_EOL . PHP_EOL . $e->getTraceAsString() . '</pre>');
```

Comment:
Toto je úplne posledná vlna obrany, tu by to nikdy nemalo skončiť (jedine ak padne error handler). Preto by som tu nevypisoval HTML